app/models/file_depot_ftp.rb

```diff
@@ -93,12 +93,15 @@ def file_exists?(file_or_directory)
   private
 
   def create_directory_structure(directory_path)
-    Pathname.new(directory_path).descend do |path|
```

Comment:
@isimluk I had explicit reasons for using descend.

See https://github.com/ManageIQ/manageiq/pull/1603

If this works on iis, or any of the many varieties of ftp servers, I'm ok with it.